packages/astro-font/utils.ts

```diff
@@ -14,9 +14,9 @@ interface Source {
   preload?: boolean
   css?: Record<string, string>
   // https://developer.mozilla.org/en-US/docs/Web/CSS/font-style
-  style: 'normal' | 'italic' | 'oblique' | `oblique ${number}deg` | GlobalValues | {}
+  style: 'normal' | 'italic' | 'oblique' | `oblique ${number}deg` | GlobalValues | (string & {})
   // https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight
-  weight?: 'normal' | 'bold' | 'lighter' | 'bolder' | GlobalValues | FontWeight | `${FontWeight}` | {}
+  weight?: 'normal' | 'bold' | 'lighter' | 'bolder' | GlobalValues | FontWeight | `${FontWeight}` | (string & {})
```

Comment:
we need the support for font weights here which aren't string but between 1 to 1000 and not divisble by 100